adrs/Enums.md

```diff
@@ -0,0 +1,25 @@
+# Enum as string in the database
+
+## Status
+Proposed
+
+## Context
+We need to specify language for the book, which we will add to the system. We have to decide how we will store this information in the database.
+
+## Alternatives
+
+### One alternative
+Create a “Language” table and add a language to the book as language_id.
+#### Pros
+- Following the third normal db form - if the data of one table changes, the data in the related tables will also change.
+#### Cons
+- There are no more than 2 languages of books at the moment and in the near;
+- Languages do not change, are not edited.
```

Comment:
But we've decided that a book's language can change, right? if I accidentally clicked on the wrong language.